cli/crates/common/src/environment.rs

```diff
@@ -30,7 +30,9 @@ pub struct Environment {
     /// the path of `$PROJECT/grafbase/schema.graphql`, the Grafbase schema,
     /// in the nearest ancestor directory with said directory and file
     pub project_grafbase_schema_path: PathBuf,
-    /// the path of `$HOME/.grafbase`, the user level local developer tool cache directory
+    /// the path of `.grafbase` folder, contains local developer tool cache.
+    /// By default placed in $HOME, if that fails or `--nohome` is supplied, will be under the project's grafbase folder.
+    // TODO: rename to dot_grafbase_path
```

Comment:
IMHO I'd rather name it by its function, `shared_cache_path` or something like that.